ladybug/src/main/java/org/frankframework/ibistesttool/DeploymentSpecificsBeanPostProcessor.java

```diff
@@ -92,6 +93,14 @@ public Object postProcessBeforeInitialization(Object bean, String beanName) thro
 				}
 			}
 		}
+		if (bean instanceof SpringLiquibase) {
+			boolean active = APP_CONSTANTS.getBoolean("ladybug.jdbc.migrator.active",
+					APP_CONSTANTS.getBoolean("jdbc.migrator.active", false));
+			if (!active) {
+				SpringLiquibase springLiquibase = (SpringLiquibase)bean;
+				springLiquibase.setShouldRun(active);
+			}
```

Comment:
Waarom zit hier een `if` omheen? Waarom zetten we dat niet gewoon altijd op de waarde van `active`?

Reply:
Scheelt het uitvoeren van wat code / een cast maar maakt het er niet leesbaarder op. En als we altijd op die waarde zetten zoals jij zegt weten we natuurlijk ook zeker dat die code werkt. Ik zal het aanpassen.

Reply:
Het zal niet veel schelen in uitvoering code en de cast moet je natuurlijk nog steeds doen maar door altijd de waarde te zetten met wat in het property staat, weet je zeker dat het property van de configuratie wordt gehonoreerd en niet wat in de Spring bean als default staat.

Ook vind ik het minder duidelijk dat er staat `springLiquibase.setShouldRun(active)` binnen de conditie `if (!active)` want dan moet ik iedere keer weer even denken "wat is nou de waarde die hier conditioneel wordt gezet"?

Conditioneel `springLiquibase.setShouldRun(false)` vind ik duidelijker, en altijd niet-conditioneel `springLiquibase.setShouldRun(active)` is voor mijzelf ook duidelijk.

Maar ja dat ben ik, een ander ziet dat misschien anders. 😉

Reply:
Volgens mij zeg je ongeveer hetzelfde als ik alleen omschrijf jij het een stuk beter. :) Ik was mijn comment begonnen met de reden waarom ik het destijds denk ik heb gedaan (weet het niet eens meer precies) waardoor je misschien dacht dat de rest van mijn comment ook redenen zouden zijn om het niet aan te passen. Dat is niet het geval. Ik ben en was het al met je eens en had het na die comment waarin ik ook zei het aan te zullen passen het ook al meteen aangepast. :)

Zie https://github.com/frankframework/frankframework/pull/6385/commits/df7024727abd560d0eea75e039e79e99ddbd4203